app/models/quiz.rb

```diff
@@ -0,0 +1,7 @@
+class Quiz < ApplicationRecord
+  belongs_to :course
+  has_many :questions, dependent: :destroy
+
+  validates :title, presence: true
+  validates :total_marks, presence: true, numericality: { only_integer: true, greater_than: 0 }
```

Comment:
why total_marks can not be float?

Reply:
It can be float apu. Currently doing it for integer values only.

Reply:
Apu, currently considering only integer marks for each questions.